dataland-frontend/src/components/general/apiKey/CreateApiKeyCard.vue

```diff
@@ -107,45 +108,45 @@ export default defineComponent({
       { label: "No expiry", value: "noExpiry" },
     ],
   }),
-  computed: {},
+  created() {
+    this.minDate = new Date(new Date().getTime() + 86400000);
+  },
```

Comment:
I think instead of this ```created()``` block you could directly calculate this value as starting value in the ```[data()](data: () =>``` block like this:

```typescript
data: () => ({
    expiryTimeDays: null as null | number,
    expiryTimeDropdown: "",
    isExpiryDateValid: true,
    minDate: new Date(new Date().getTime() + 86400000),
    customDate: null,
    days: [
      { label: "7 days", value: 7 },
      { label: "30 days", value: 30 },
      { label: "60 days", value: 60 },
      { label: "90 days", value: 90 },
      { label: "Custom...", value: "custom" },
      { label: "No expiry", value: "noExpiry" },
    ],
  }),
````

Reply:
done